src/mui/list/List.js

```diff
@@ -102,10 +106,9 @@ export class List extends Component {
         return this.props.location.pathname;
     }
 
-    refresh = (event) => {
+    refresh = event => {
         event.stopPropagation();
-        this.fullRefresh = true;
-        this.updateData();
```

Comment:
It looks like 'isLoading' state change when references are loaded. Therefore the list renders when 'isLoading' has been changed.
![2017-05-02_18-54-51](https://cloud.githubusercontent.com/assets/2131624/25626752/32011a04-2f69-11e7-9a4c-fc75d902ee9a.png)